reconcile/utils/slack_api.py

```diff
@@ -392,7 +392,19 @@ def get_users_by_names(self, user_names: Iterable[str]) -> dict[str, str]:
         }
 
     def get_users_by_ids(self, users_ids: Iterable[str]) -> dict[str, str]:
-        return {k: v["name"] for k, v in self._get("users").items() if k in users_ids}
+        return {
+            user_id: user["name"]
+            for user_id in users_ids
+            if (user := self._get("users").get(user_id))
```

Comment:
TIL: wow you can use the walrus operator here as well. 👍

Reply:
reference here https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions

Reply:
nice